modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -3,263 +3,388 @@ ifndef::env-site,env-github[]
 
-=== Test the Design Center Application
+To test your app:
 
-. Open a command line and enter this command:
+. Click the copy icon to copy the deployment URL:
 +
-[source,console]
+.The arrow shows the deployment URL copy icon.
+image::os-tutorial-dc-copy-link.png[The copy deployment URL icon]
+. Run the following `curl` command from a command prompt to send the key-value pair. 
++
+Replace _DEPLOYMENT-URL_ with the deployment URL you copied from Design Center.
++
+[source,json,linenums]
 ----
-curl -X POST -H "Content-Type: application/json" -d '{ "key": "MyKey", "value": "OSv2 Test" }' "http://LINK_FROM_DESIGN_CENTER/store"
+
+curl -X POST \
+  "http://DEPLOYMENT-URL/store"
+  -H "Content-Type: application/json" 
+  -d '{ "key": "myKey", "value": "OSv2 Test" }' 
+
 ----
 +
-. Substitute `LINK_FROM_DESIGN_CENTER` with the link you copied from Design Center, and then press Enter to send the command to Design Center.
-+
 The output is an echo:
 +
 [source,console]
 ----
-{ "key": "MyKey", "value": "OSv2 Test" }
+{ "key": "myKey", "value": "OSv2 Test" }
 ----
+. Select *View in Runtime Manager* from the *...* menu to the right of the *Test* button:
 +
-. In Anypoint Platform, click the menu icon in the upper left and click *Runtime Manager*.
-. Notice how an entry has been created for you in the list of applications.
-. Click *CloudHub* in the Server column for your application to display details, and
-click *Manage Application*.
+.The arrow shows the *View in Runtime Manager* option in the *...* menu in Design Center.
+image::os-tutorial-dc-view-rtm-option.png[View in Runtime Manager menu option]
+. In *Runtime Manager*, click *Object Store* in the left navigation pane.
+. Click the object store, the partition, and then the key:
 +
-image::os-rm-manage-app.png[]
+.The screenshot shows (*1*) the Object Store, (*2*) the key, and (*3*) the value in the Object store.
+image::os-tutorial-rtm-key-in-os.png[Key and value in the Object Store in Runtime Manager]
 +
-. Click *Object Store* and click the object store, then the partition, and the key. You can
-see the value as `[binary value] BINARY`.
+You see that the myKey key that you sent from curl appears in the Object Store with the value of [binary value] BINARY.
++
+*DO WE NEED THIS DETAIL?*
++
+The `myKey` key that you sent from `curl` appears in the Object Store with the value of `[binary value] BINARY`.
+The value is binary because Mule 4 wraps values in a Mule object, which causes
+them to be visible only as binary in Anypoint Platform.
+Behind the scenes, Mule 4 executes binary serialization with the Mule internal serializer.
+The user interface cannot deserialize the object, so it shows that the value is now binary.
+
+You have successfully tested your app using a `curl` command to store a key-value pair in the object store and viewed the results in Runtime Manager.
+
+To verify that the value is correct, return to Design Center and add a new flow that retrieves the value.
```

Comment:
Ditto edit comments made at the completion of the previous task, re. "You have successfully" etc.

Reply:
BREAK EDIT HERE